src/components/ProblemsTable.scss

```diff
@@ -11,3 +11,7 @@
   gap: var(--iui-size-xs);
   justify-content: space-between;
 }
+
+:where(.iui-root, [data-iui-theme]) .iui-table-body .iui-table-row.active .iui-table-cell {
```

Comment:
Please avoid targeting internal iui- classes. You can apply your custom className to a `<DefaultCell>` passed to `cellRenderer`. Or use a custom class in `rowProps` then target the cell using `> *`.